pilot/cmd/cmd.go

```diff
@@ -49,20 +50,33 @@ var VersionCmd = &cobra.Command{
 	},
 }
 
-// AddFlags carries over glog flags with new defaults
-func AddFlags(rootCmd *cobra.Command) {
+// InitRootCommand Initializes the root command with logging configuration
+func InitRootCommand(rootCmd *cobra.Command, loggingOptions *log.Options) {
```

Comment:
@kyessenov please review the changes here for configuring istio logging and glogs